src/style.css

```
@@ -1,3 +1,64 @@
-body {
-  margin: 0;
+:root {
+  --main-text-color: #000;
+  --edit-text-color: #00ACDC;
 }
+
+  html, body {
+    margin: 0;
+    font-family: Roboto, sans-serif;
+  }
```

html doesn't have default margin
```suggestion
  html {
    font-family: Roboto, sans-serif;
  }
  
  body {
    margin: 0;
  }
```